src/components/NamespaceCard.js

```diff
@@ -89,12 +89,12 @@ const NamespaceCard = ({
   }, [projectId]);
 
   function containsSpecialChars(str) {
-    const specialChars = /[`!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~]/;
+    const specialChars = /[`!@#$%^&*()_+\=\[\]{};':"\\|,.<>\/?~A-Z]/;
```

Comment:
rather should we check it against the valid regex which is '[a-z0-9]([-a-z0-9]*[a-z0-9])?'

Reply:
updated with `^[a-z0-9]([-a-z0-9]*[a-z0-9])?$`